internal/helmutil/index_v2.go

```diff
@@ -121,6 +121,10 @@ func (idx *IndexV2) SortEntries() {
 	idx.index.SortEntries()
 }
 
+func (idx *IndexV2) UpdateGeneratedTime() {
+	idx.index.Generated = time.Now()
```

Comment:
```suggestion
	idx.index.Generated = time.Now().UTC()
```

Let's do this so we're not relying on OS timezone.